.env.example

```
@@ -2,4 +2,7 @@
 ALCHEMY_API_KEY=your_alchemy_api_key_here
 
 # Etherscan API Key for contract verification
-ETHERSCAN_API_KEY=your_etherscan_api_key_here
\ No newline at end of file
+ETHERSCAN_API_KEY=your_etherscan_api_key_here
+
+# Chainlink VRF Subscription ID (from vrf.chain.link/sepolia)
+VRF_SUBSCRIPTION_ID=your_subscription_id_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)